extensions/aria_extension_tosca/simple_v1_0/types.py

```diff
@@ -13,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from aria.utils.collections import FrozenDict, FrozenList
+from aria.utils.collections import (FrozenDict, FrozenList)
```

Comment:
I thought we agreed that that kind of style is a recommendation but not a requirement. If we switch to that style here it would make a lot of the parser code extremely verbose and frankly unreadable.